File: src/cli/tests/fixtures/doc/python/doc_text_lexicon_budgets.py
# Loads the configured data from disk, parses every record against the
# schema, resolves relative paths, retries transient failures with a
# bounded backoff policy, and logs a one-line summary once the load
# settles into a steady state that operators can rely on every time.

payload = """
# not a comment: this triple-quoted string line runs well past eighty
# characters, and these payload lines pool far past the two hundred
# forty paragraph budget when joined, so a scanner that mistook the
# string for comments would fire both text checks on all these lines
"""

mask = 1 << 4

# A short comment.
def quiet(payload, mask):
    return payload, mask

File: src/cli/tests/fixtures/doc/doc_text_lexicon_budgets.rb
# Loads the configured data from disk, parses every record against
# the schema, resolves relative paths, retries transient failures,
## heading-shaped prose lines still measure as prose rather than pass
# silently through any heading exemption borrowed from markdown files.

list = []
list << :item

text = <<~PAYLOAD
# not a comment: this heredoc payload line runs well past the eighty
# character budget, and the payload lines pool far past the paragraph
# budget when joined, so a scanner that mistook the payload for
# comments would fire both text checks on all of these payload lines
PAYLOAD

def quiet(list, text)
  [list, text]
end
